src/status_im2/contexts/wallet/common/utils.cljs

```diff
@@ -43,3 +43,15 @@
               (* (total-token-value-in-all-chains token)
                  (-> token :market-values-per-currency :usd :price))))
        (reduce +)))
+
+(defn calculate-fiat-change
+  [fiat-value change-pct-24hour]
+  (* fiat-value (/ change-pct-24hour (+ 100 change-pct-24hour))))
+
+(defn get-balance-by-address
+  [balances address]
+  (->> balances
+       (filter #(= (:address %) address))
+       first
+       :balance))
```

Comment:
Three things:

1. I think this function is not being used, I'm not sure.
2. Instead of using `filter` + `first`, we can just use `some`.
3. This one should be a sub depending on `:wallet/balances`

Reply:
Yes it is not being used. Removed it 👍